docs/releases/1.10.txt

```diff
@@ -161,6 +161,9 @@ Management Commands
   command exit with a non-zero status when model changes without migrations are
   detected.
 
+* The Web server launched by the :djadmin:`runserver` command claims HTTP 1.1
```

Comment:
It looks to me (no expert either), like we'd also need to set [protocol_version](https://docs.python.org/3.5/library/http.server.html?highlight=protocol_version#http.server.BaseHTTPRequestHandler.protocol_version) on the `WSGIServer` too. Not sure this is trivial given the comment about `Content-Length` in the Python docs of that attribute.